### PR TITLE
Add fma to Math docs index

### PR DESCRIPTION
**Pull request by jabraham17**
Adds the newly added `fma` function to the list of Math functions at the top of the Math module.

Tested `test/library/standard/Math/docs` locally

[Reviewed by @lydia-duncan]

### CODE REVIEW

File: modules/standard/Math.chpl [hunk 1]
```diff
@@ -155,6 +155,12 @@ Bessel Functions
 :proc:`~Math.y1`
 :proc:`~Math.yn`
 
+.. _math-other:
+
+Remaining Functions
```
Comment (lydia-duncan) [inline modules/standard/Math.chpl:160]: This is definitely fine as is, but I wouldn't be opposed to calling it "Optimizations" or something along those lines.

**Review by lydia-duncan — APPROVED** [review]
Thanks!  And also, thanks for adding the check to ensure everything is listed, we definitely would have gotten out of date without it and not noticed :)

File: modules/standard/Math.chpl (hunk 1 above)
Comment (jabraham17) [reply modules/standard/Math.chpl:160]: What do you think about calling it "Builtins"?

Comment (lydia-duncan) [reply modules/standard/Math.chpl:160]: To me, that would imply that it should be defined in AutoMath.  Which we may still do eventually, but I don't think is the right call while it is still prototypical

**Pull request merged**